hydromt/models/model_network.py

```diff
@@ -73,21 +77,22 @@ def write(
         Parameters
         ----------
         components : List, optional
-            List of model components to write, each should have an associated write_<component> method.
-            By default ['config', 'maps', 'network', 'geoms', 'forcing', 'states']
+            List of model components to write, each should have an
+            associated write_<component> method. By default ['config', 'maps',
+            'network', 'geoms', 'forcing', 'states']
         """
         super().write(components=components)
 
     # TODO: make NetworkMixin class with following properties/methods
     @property
-    def network(self):
+    def network(self):  # noqa: D102
```

Comment:
what does  # noqa: D102 mean?

Reply:
This tells the linter to ignore rule D102 (i.e. public methods need to have a doc string) for this line only. Since these are abstract methods I did not think a doc string was really necessary.